src/SFA.DAS.AssessorService.Web/Startup.cs

```diff
@@ -85,18 +85,11 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                 services.AddTransient<ICustomClaims, AssessorServiceAccountPostAuthenticationClaimsHandler>();
                 services.AddTransient<IStubAuthenticationService, StubAuthenticationService>();
                 
-                if (Configuration.UseGovSignIn)
-                {
-                    var isLocal = string.IsNullOrEmpty(_config["ResourceEnvironmentName"])
-                                  || _config["ResourceEnvironmentName"].Equals("LOCAL", StringComparison.CurrentCultureIgnoreCase);
-                    var cookieDomain = isLocal ? "" : Configuration.ServiceLink.Replace("https://", "", StringComparison.CurrentCultureIgnoreCase);
-                    var loginRedirect = isLocal ? "" : $"{Configuration.ServiceLink}/service/account-details";
-                    services.AddAndConfigureGovUkAuthentication(_config, typeof(AssessorServiceAccountPostAuthenticationClaimsHandler), "/account/signedout", "/service/account-details", cookieDomain, loginRedirect);
-                }
-                else
-                {
-                    services.AddAndConfigureAuthentication(Configuration, _env);
```

Comment:
I don't see AddAndConfigureAuthentication being removed - wouldn't it now be redundant?

Reply:
I think I've removed the else clause that calls AddAndConfigureAuthentication, and left in the ...GovUk... equivalent. Do you mean I should remove the method definition from the AuthenticationStartup class?

Wouldn't that mean the whole AS class is now redundant?